src/VaultStore/VaultStore.ts

```diff
@@ -0,0 +1,52 @@
+import Vault from "@polykey/VaultStore/Vault";
+
+class VaultStore {
```

Comment:
This should be better represented with capability theory.

Reply:
Might not need to be anymore...

Reply:
Well we could remove the vault store and just store which pubkeys have access to the vault in the vault class itself.

This is also related to a thread above:
> If VaultStore is for storing relationships between public keys and vaults, then this is a strange name for it. I'd prefer that we talk little about capability theory.